src/pip/_internal/cli/cmdoptions.py

```diff
@@ -509,12 +535,13 @@ def _handle_python_version(option, opt_str, value, parser):
     action='callback',
     callback=_handle_python_version, type='str',
     default=None,
-    help=("Only use wheels compatible with Python "
-          "interpreter version <version>. If not specified, then the "
-          "current system interpreter minor version is used. A major "
-          "version (e.g. '2') can be specified to match all "
-          "minor revs of that major version.  A minor version "
-          "(e.g. '34') can also be specified."),
+    help=dedent("""\
+    The Python interpreter version to use for wheel and "Requires-Python"
+    compatibility checks. Defaults to a version derived from the running
+    interpreter. The version can be specified using up to three dot-separated
+    integers (e.g. "3" for 3.0.0, "3.7" for 3.7.0, or "3.7.3"). A major-minor
+    version can also be given as a string without dots (e.g. "37" for 3.7.0).
+    """),
```

Comment:
I thought we'd want to deprecate the `37` values in favor of the unambiguous `3.7` ?

Reply:
That can still be done, but I was hoping it could be done in a separate PR. I usually prefer keeping PR's smaller when possible to make it easier for people to review and discuss changes, etc (and code them :) ).

Reply:
Several PRs are fine by me